src/main/java/com/team973/frc2025/subsystems/Elevator.java

```diff
@@ -123,6 +114,10 @@ private TalonFXConfiguration defaultElevatorMotorConfig() {
     return defaultElevatorMotorConfig;
   }
 
+  public Pose3d getPose() {
+    return new Pose3d();
```

Comment:
Let's do a real implementation here based on where the robot thinks the elevator is IRL --- something like `new Pose3d(0, 0, m_heightInMeters())`

If we log what the robot thinks its position is during a match then I think we can render all that when we review the logs